force-app/main/default/classes/SOQL.cls

```diff
@@ -39,9 +40,11 @@ public with sharing class SOQL {
      * @return A list of Contact records.
      * Update the corresponding test class and testGetContactsByFirstName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Contact> getContactsByFirstName () {
+        String name = 'John';
+        List<Contact> conts = [SELECT Id, Name FROM Contact WHERE FirstName = :name];
+        return conts;
+    }
```

Comment:
How can we write this method so that it takes any String name and finds the firstName for that method? Currently it will only find 'John'.